<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>375</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
    <weight>9</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gui</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 0, 255, 255), stop:0.970588 rgba(0, 0, 0, 255));
font: 75 10pt &quot;Times New Roman&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Files</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListView" name="fileOneListView">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="fileTwoListView">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Clear Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Atom Selection</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Atoms">
                <property name="styleSheet">
                 <string notr="true">color: rgb(20, 0, 26);
</string>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContents</enum>
                </property>
                <item>
                 <property name="text">
                  <string>BackBone</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Sidechain</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>All</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Residue Range 1</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLineEdit" name="R1Start"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="R1End"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Residue Range 2</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLineEdit" name="R2Start"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="R2End"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Operation</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="operation">
                <property name="styleSheet">
                 <string notr="true">color: rgb(20, 0, 26);</string>
                </property>
                <item>
                 <property name="text">
                  <string>Nothing</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Translation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Rotation</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Translation and Rotation</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Report</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="flag">
                <property name="styleSheet">
                 <string notr="true">color: rgb(20, 0, 26);</string>
                </property>
                <item>
                 <property name="text">
                  <string>No Printout</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RMSD per Residue</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RMSD per Atom</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dihedral Angles in Radians</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dihedral Angles in Degrees</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Write PDB</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="printBox">
                <property name="styleSheet">
                 <string notr="true">color: rgb(40, 0, 30);</string>
                </property>
                <item>
                 <property name="text">
                  <string>No</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Yes</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetNoConstraint</enum>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>RMSD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="RMSD">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="calcRMSD">
              <property name="text">
               <string>Calculate RMSD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="File1Show">
              <property name="text">
               <string>File 1 Display</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="File2Show">
              <property name="text">
               <string>File 2 Display</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="Display">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExport_Angles"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExport_Angles">
   <property name="text">
    <string>Export Angles</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
